src/Microsoft.AspNet.Server.Kestrel/Http/ListenerSecondary.cs

```diff
@@ -59,7 +59,7 @@ protected ListenerSecondary(ServiceContext serviceContext) : base(serviceContext
 
                                 DispatchPipe.ReadStart(
                                     (_1, _2, _3) => buf,
-                                    (_1, status2, error2, state2) =>
+                                    (_1, status2, errCode, error2, state2) =>
```

Comment:
wtf is _1, status2, errCode, erro2, state2? I know we have nested lambdas by maybe this is a case for a method...